cart/domain/cart/cartServicePorts.go

```diff
@@ -77,9 +78,16 @@ type (
 		// Source Id of where the items should be initial picked - This is set by the SourcingLogic
 		SourceID       *string
 		Qty            *int
+		ItemID         string
```

Comment:
We should at least add a comment here that ItemID and DeliveryCode are used for finding the item in the cart and therefore are mandatory. the fields are not for changing the items for example the delivery code of item xy

Reply:
Since we anyhow change the interface..

Didn't we decide a while ago that itemId is the unique idendifier of an item and should be unique for complete cart?

Reply:
True maybe we then just use the ItemID here. Maybe we add a function to get the delivery of an item in case that the underlying adapter needs the deliverycode

Reply:
Something like cart.GetDeliveryByItem(..)